src/components/include/transport_manager/transport_manager.h

```diff
@@ -227,6 +227,13 @@ class TransportManager {
    * @return device info for WebEngine device
    */
   virtual const DeviceInfo& GetWebEngineDeviceInfo() const = 0;
+
+  /**
+   * @brief Checks if such an application is in any of the connections.
+   * @param id Application's connection uid.
+   * @return True if the application present in the any connections.
```

Comment:
@ydementieiev `if the application present in the any connections.` -> `if the application is present in connections list`

Reply:
@AByzhynar changed, thank you